README.md

```diff
@@ -11,26 +23,101 @@ _This project is currently in its alpha stage and might be incomplete or change
 pip install changelist
 ```
 
-## Set up your repository
+## Usage
+
+The script requires a [GitHub personal access
+token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens).
+The token does not need any permissions, since it is used only to
+increase query limits.
+
+```sh
+export GH_TOKEN='...'
+changelist scientific-python/changelist v0.2.0 main
+```
 
-To categorize merged PRs in the changelist, each PR
-must have have one of the following labels:
+This will list all pull requests, authors and reviewers that touched commits
+between `v0.2.0` and `main` (excluding `v0.2.0`).
+Pull requests are sorted into section according to the configuration in
+`tool.changelist.label_section_map`.
+
+## Configuration
+
+changelist can be configured from two sources, in order of precedence:
+
+- A local TOML file specified with the `--config` option
+- A remote `pyproject.toml` at `stop_rev`
+
+If a configuration option is not specified in either file above, changelist
+falls back to the following configuration:
+
+````toml
```

Comment:
Could this be somehow sourced into the README from the actual reference? To lower the chance of diverging the docs from the actual behaviour.

Reply:
I will merge and follow-up with a PR to (hopefully) address your suggestions.